stdlib/SparseArrays/test/sparse.jl

```diff
@@ -2359,4 +2359,51 @@ end
     @test one(A) isa SparseMatrixCSC{Int}
 end
 
+@testset "circshift" begin
+    for i=1:20
+        m,n = 17,15
+        A = sprand(m, n, rand())
+        shifts = rand(-m:m), rand(-n:n)
```

Comment:
I would make this deterministic to make sure we always exercise the corner cases.
```jl
    for rshift in (-1, 0, 1, 10), cshift in (-1, 0, 1, 10)
        shifts = (rshift, cshift)
```

Reply:
Thanks @stevengj. I've done it. I moved the sparse vector tests to test/sparsevector.jl (which already existed).